packages/medusa/src/loaders/config.ts

```diff
@@ -1,9 +1,30 @@
+import { getConfigFile } from "medusa-core-utils"
+import resolveCwd from "resolve-cwd"
 import { ConfigModule } from "../types/global"
-import { getConfigFile } from "medusa-core-utils/dist"
 import logger from "./logger"
 
 const isProduction = ["production", "prod"].includes(process.env.NODE_ENV || "")
 
+type ModuleDefinition = {
+  registration: string
+  defaultPackage: string
+  label: string
+  validation: (proto: any) => boolean
+  required: boolean
+  canOverride: boolean
+}
+
+export const MODULE_DEFINITION: Record<string, ModuleDefinition> = {
```

Comment:
suggestion: Should we move that to another place to not have a lot of definition ending up in the config loader? maybe under modules.ts a bit like the feature flags?